libs/blocks/share/inline-utils.js

```diff
@@ -0,0 +1,18 @@
+import { createTag } from '../../utils/utils.js';
+
+export function inlineBlock(block) {
+  if (!block.classList.contains('inline')) return;
+
+  const section = block.closest('.section');
+  const inlineSiblinggs = section.querySelectorAll('.inline');
+  if (inlineSiblinggs.length > 1) {
```

Comment:
If there are multiple inline blocks in a section that are not next to each other, they will be moved to a new order and placed next to each other.  See:

https://share-block-updates--milo--adobecom.hlx.page/drafts/cpeyer/bleh

![Screenshot 2023-03-20 at 10 13 17 AM](https://user-images.githubusercontent.com/9143715/226366655-4d12c5df-dbe9-4a04-8734-41c3a9298943.png)

Where the 2 inline blocks are in different parts of the section.  Are we ok with merging into one location?

Reply:
Alright, it should be fixed now: 

https://share-block-updates--milo--adobecom.hlx.page/drafts/cpeyer/bleh

https://main--stock--adobecom.hlx.page/drafts/solene/share-reading-time-no-sections?martech=off&milolibs=share-block-updates

https://main--stock--adobecom.hlx.page/drafts/solene/share-reading-time?martech=off&milolibs=share-block-updates